### PR TITLE
Prepare examples of prettier formatted code

### PR DESCRIPTION
**Pull request by JuliaGerasymenko**

### CODE REVIEW

**Review by tshemsedinov — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -273,7 +273,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```
Comment (lundibundi) [inline package-lock.json]: These must be reverted back to https.

File: package.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
   ],
   "scripts": {
     "test": "eslint . && metatests test",
```
Comment (lundibundi) [inline package.json]:
```suggestion
     "test": "npm run lint && metatests test",
```

**Review by SemenchenkoVitaliy — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (belochub) [reply package.json]:
```suggestion
     "test": "npm run -s lint && metatests test",
```

File: package-lock.json [hunk 3]
```diff
@@ -387,7 +387,7 @@
         },
         "util": {
           "version": "0.10.3",
-          "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
+          "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
```
Comment (lundibundi) [inline package-lock.json]: This is still not resolved, could you revert `http` back to `https` here?

**Comment by lundibundi** [comment]
Landed in https://github.com/metarhia/filestorage/commit/f9e3d0d5ce2e94849a6e0a161f64158bda45240e.

**Pull request closed without merging**